packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -379,6 +379,9 @@ class FlutterErrorDetails with Diagnosticable {
   /// The [exception] must not be null; other arguments can be left to
   /// their default values. (`throw null` results in a
   /// [NullThrownError] exception.)
+  // If you add a property to this class, do not forget to update _runTest in
```

Comment:
It might be better to implement a copyWith instead of forcing us to go into flutter_test to make changes.

Reply:
Done